### ISSUE TITLE
🐛 Biome Not Linting NestJs Repositories

### ISSUE THREAD

**Issue by vbasky** [biomejs/biome-vscode#162]
### VS Code version

1.87

### Extension version

2.2.1

### Biome version

1.5.3

### Operating system

- [ ] Windows
- [X] macOS
- [ ] Linux

### Description

Attached is the screenshot of the NestJs controller and what Biome is complaining about

<img width="750" alt="image" src="https://github.com/biomejs/biome-vscode/assets/287302/5633e500-8312-4ec9-bf81-5d39f8db93b8">

`biome.json` settings as below

```
 "javascript": {
    "formatter": {
      "quoteStyle": "single",
      "trailingComma": "none"
    },
    "parser": {
      "unsafeParameterDecoratorsEnabled": true
    }
  } 
```

### Steps to reproduce

1. Open any Nestjs default project
2. Add biome.json and dependency
3. Add @Param under a @Get or @Post call

Editor starts complaining

### Expected behavior

VSCode extension is not respecting the rules that CLI is respecting

### Does this issue occur when using the CLI directly?

No

**Comment by nhedger** [comment]
Hey @vbasky,

Is there any chance that your `biome.json` file is not at the root of your workspace folder?

As you can see in the following video, the diagnostic appears when the `biome.json` file is not at the root of the workspace folder, causing it to fallback to the defaults, in which `unsafeParameterDecoratorsEnabled`is set to `false`.



https://github.com/biomejs/biome-vscode/assets/649677/3f74bbd8-070b-4f34-8580-f801cddefd7d

**Comment by vbasky** [comment]
@nhedger 

Thank you that was the issue we had an app folder and the biome.json was within it. That solves it for now but is there a way to configure it within a folder ?

**Comment by nhedger** [comment]
Thanks for confirming @vbasky.

Unfortunately, there isn't support for this in the extension right now.
